corehq/apps/app_manager/migrations/0025_migrate_to_search_filter_flag.py

```diff
@@ -0,0 +1,26 @@
+import traceback
+from django.db import migrations
+
+from django.core.management import call_command
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def migrate_to_search_filter_flag(apps, schema_editor):
+    try:
```

Comment:
It's typical to let `call_command` calls like this fail hard. It will block deploy, but if it fails, something is genuinely wrong. It's too easy for a deployer to miss messages during deploy, between the amount of output and the confusion of output from different machines getting mixed together.

Reply:
Ahh, I wasn't sure, but that definitely makes sense.

Reply:
After talking with @calellowitz about it (since he seemed to have dealt with similar issues in the past), I added a try/except block on `get_current_app` only, to handle some broken apps on staging that were throwing `WrappingAttributeError` or `BadValueError`.  That function fails if it encounters an app doc that is not able to be wrapped, and per Cal if you can't wrap an app, you can't use it - so it should be safe to ignore if any come up in this migration. Though for what it's worth, there don't appear to be any apps this migration would touch on production that have that issue.

Reply:
If an app throws an error on being wrapped, it's inaccessible in HQ, since all of the app builder pages will 500. It's typically an issue for old apps that probably aren't in use anymore.